mvn-extension/src/test/java/org/arquillian/smart/testing/mvn/ext/dependencies/SurefireApiDependencyTest.java

```diff
@@ -56,10 +60,73 @@ public void should_not_add_surefire_dependency_as_it_is_present()
         assertThat(actualVersion).isEqualTo("2.19.1");
     }
 
+    @Test
+    public void should_autocorrect_strategy()
+        throws Exception {
+        // given
+        Model model = prepareModelWithSurefirePlugin("2.20");
+        final Configuration conf = Configuration.load();
+        conf.setAutocorrect(true);
+        conf.setStrategies("nwe");
+        final DependencyResolver dependencyResolver = new DependencyResolver(conf);
+        model.addDependency(new DependencyResolver.SurefireApiDependency("2.19.1"));
```

Comment:
Extract this whole given part to separated method - the same code is used in other test methods

Reply:
Why this didn't happen?